worlds/ror2/options.py

```diff
@@ -375,6 +375,44 @@ class ItemWeights(Choice):
     option_void = 9
 
 
+ror2_option_groups = [
+    OptionGroup("Explore Mode Options", [
+        ChestsPerEnvironment,
+        ShrinesPerEnvironment,
+        ScavengersPerEnvironment,
+        ScannersPerEnvironment,
+        AltarsPerEnvironment,
+        RequireStages,
+        ProgressiveStages,
+    ]),
+    OptionGroup("Classic Mode Options", [
+        TotalLocations,
+    ], start_collapsed=True),
+    OptionGroup("Weighted Choices", [
+        ItemWeights,
+        ItemPoolPresetToggle,
+        GreenScrap,
+        RedScrap,
+        YellowScrap,
+        WhiteScrap,
```

Comment:
```suggestion
        WhiteScrap,
        GreenScrap,
        RedScrap,
        YellowScrap,
```
To match the order of the items below

Reply:
Thanks. Fun fact (just learned), the order doesn't matter in the group. It goes off of the dataclass order